backside/places/handler.go

```diff
@@ -0,0 +1,36 @@
+package places
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"sk8.town/backside/places/dto"
+)
+
+type Handler struct {
+	Service Service
+}
+
+func (handler Handler) GetPlacesAutocomplete(writer http.ResponseWriter, request *http.Request) {
+	var placesRequest dto.AutocompleteRequest
+	err := json.NewDecoder(request.Body).Decode(&placesRequest)
+	if err != nil {
+		writeResponse(writer, http.StatusBadRequest, err.Error())
```

Comment:
In case of `400` we still return a json content type and ugly `EOF` message:
<img width="1231" alt="image" src="https://user-images.githubusercontent.com/85256521/130154155-0f3aed1b-8bbf-447d-a06a-669644eb5482.png">

I think we should have something like `writeErrorResponse` or similar 🤷